aten/src/ATen/function_wrapper.py

```diff
@@ -999,11 +1001,17 @@ def emit_body(env, option):
         elif ret['kind'] == 'type':
             assert len(calls) == 1
             call = calls[0]
-            if ret['type'] == 'THTensor*':
+            return_tensor_map = {
```

Comment:
I think this list should be merged with ALLOC_WRAP, which serves a similar purpose.
ALLOC_WRAP  currently doesn't take an ${arg_name} parameter, but it can added as an option:

Example:
```
 'THTensor*': 'new ${Tensor}(context${,arguments})',
```

Then when formatting you can set `CodeTemplate(ALLOC_WRAP[key]).format(arguments=[])` to leave arguments blank or set it `CodeTemplate(ALLOC_WRAP[key]).format(arguments=[call])` when you want to add add something to the list.